tests/Integration/ScriptsTest.php

```diff
@@ -402,4 +402,50 @@ public function assert_script_statuses( string $handle, array $assert_true = arr
 			);
 		}
 	}
+
+	/**
+	 * Test if the web stories analytics got enqueued.
+	 *
+	 * @since 3.2.0
+	 *
+	 * @covers \Parsely\Scripts::run()
+	 * @group scripts
+	 */
+	public function test_web_stories_script_is_enqueued(): void {
+		self::$scripts->run();
+		self::assertSame(
+			10,
+			has_action( 'web_stories_print_analytics', array( self::$scripts, 'render_amp_analytics_tracker' ) )
+		);
+	}
+
+	/**
+	 * Test if the AMP tracker render outputs the correct script.
+	 *
+	 * @since 3.2.0
+	 *
+	 * @covers \Parsely\Scripts::render_amp_analytics_tracker
+	 * @group scripts
+	 */
+	public function test_render_amp_analytics_tracker(): void {
+		ob_start();
```

Comment:
Don't use output buffering in the test - the TestCase already has this built-in - see https://phpunit.readthedocs.io/en/9.5/writing-tests-for-phpunit.html#testing-output and use `expectOutputString()` instead.